srcpkgs/cw/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'cw'
+pkgname=cw
+version=4.1.1
+revision=1
+build_style=go
+go_import_path=github.com/lucagrulla/cw
+short_desc="Best way to tail AWS CloudWatch Logs from your terminal"
+maintainer="akhil <akhil.j@outlook.in>"
+license="Apache-2.0"
+homepage="https://www.lucagrulla.com/cw"
+changelog="https://github.com/lucagrulla/cw/wiki/Release-notes"
+distfiles="https://github.com/lucagrulla/${pkgname}/archive/refs/tags/v${version}.tar.gz"
```

Comment:
expand $pkgname here. there's no benefit; it's much longer than the name itself

Reply:
I checked some other templates and they used it. so i just followed it.

Reply:
let's not add more templates that use it

Reply:
done. modified template.